.gitignore

```diff
@@ -6,3 +6,5 @@ dbuild-mp
 rbuild-mp
 .*~
 cmake/ConfigUser.cmake
+share/coast
+share/dcw
```

Comment:
@joa-quim could you also add `**/gmt.history ` to ignore any stray history files?